### DISCUSSION TITLE
Very low precision comparing with manual coding in sklearn

### DISCUSSION THREAD

**Discussion by Mjuve360** [pycaret/pycaret#2121] [Q&A] [answered]
Dear data scientists
I have a classification problem with pycaret. I do the same configs with the same data in pycaret and manual coding with sklearn. don't know why there is a huge difference in precision! what I see is that pycaret always gives me a very low precision. is there any reason for this?

**Comment by ngupta23** [comment, marked as answer]
There are many ways to structure the workflow in sklearn and pycaret. Without more details **(detailed reproducible code and data)**, it will be hard to provide specifics.

**Reply by Mjuve360** [reply]
Dear Nikhil Gupta
Here is my code file. appreciate your attention. 
[Archive.zip](https://github.com/pycaret/pycaret/files/7997565/Archive.zip)
I also attached related data in below zip file.

**Reply by ngupta23** [reply]
The data file seems to be corrupted. Can you double check?
